package.json

```diff
@@ -38,6 +38,7 @@
     "dagre": "^0.8.5",
     "konami-code": "^0.2.1",
     "react-custom-scrollbars": "^4.2.1",
+    "react-flip-toolkit": "^7.0.7",
```

Comment:
Is this dependency needed i.e. could we do it in vanilla JS (or does this just make it super trivial to implement?) - thinking build size, giving performance over to another library etc.

Reply:
I hate to introduce a new library unnecessarily, but this takes something that'd be pretty complicated to implement, and makes it trivial. i think it's worthwhile.

Reply:
Cool 👍 Any idea on the build size increase from this or is it minimal?

Reply:
22K (gzipped 7K)

It's also performing some work in lieu of `react-transition-group`, so it's got that going for it, which is nice

Reply:
👍 Nice